### DISCUSSION TITLE
Edit sound via bluetooth ?

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#679] [Q&A] [answered]
Is it possible to edit sound via an iPad or a PC without cable ?

**Comment by probonopd** [comment, marked as answer]
Unfortunately not. To implement Bluetooth would require Circle to support it, which currently is not the case:
https://github.com/rsta2/circle/blob/749fe99301846217439a40ac7bda547a986db0f7/README.md?plain=1#L129

**Reply by diyelectromusic** [reply]
I am now wondering about some kind of RF coprocessor setup to link to MiniDexed MIDI though...

There are wireless (serial) MIDI adaptors that could be used "off the shelf" or it might be possible to use a microcontroller as a some-kind-of-RF-to-serial-or-USB-MIDI converter...  The Pico and ESP32 iirc can do Bluetooth...

I'll have a think :)

Kevin

**Reply by probonopd** [reply]
Well. Wouldn't WLAN be the natural choice since we can use the built-in WLAN on RPi models that have it?

Could possibly be ported "easily" from https://github.com/dwhinham/mt32-pi/wiki/Networking:-RTP-MIDI-%28AppleMIDI%29, and could be made even better thanks to https://github.com/rsta2/circle/discussions/427#discussioncomment-9693322...

**Reply by diyelectromusic** [reply]
I'll have to have a look... :)

**Reply by diyelectromusic** [reply]
By the way... just out of interest, I've just found this: https://github.com/rppicomidi/ble-midi2usbhost

Don't know if that is of interest or not.  It uses a Pico as a BLE MIDI to USB MIDI link.

Kevin
